core/src/test/scala/org/apache/spark/benchmark/BenchmarkBase.scala

```diff
@@ -49,7 +49,11 @@ abstract class BenchmarkBase {
       val resultFileName =
         s"${this.getClass.getSimpleName.replace("$", "")}$jdkString$suffix-results.txt"
       val prefix = Benchmarks.currentProjectRoot.map(_ + "/").getOrElse("")
-      val file = new File(s"${prefix}benchmarks/$resultFileName")
+      val dir = new File(s"${prefix}benchmarks/")
+      if (!dir.exists()) {
+        dir.mkdirs()
+      }
+      val file = new File(s"${dir}$resultFileName")
```

Comment:
@byungsoo-oh can you add an assert here with a pretty error message saying the `benchmarks` directory has to be manually created? My concern is that it happened to create `benchmarks` directly in a weird location mistakenly (because the directories are based on the jar location which is pretty flaky and hacky)